src/fetching/patchQuery.ts

```diff
@@ -0,0 +1,20 @@
+import { defaultFetch } from '.';
+import { ZetkinQuery } from 'types/zetkin';
+
+const patchQuery = (orgId : string, targetId: number, fetch = defaultFetch) => {
```

Comment:
The _target_ terminology is not correct for this context. Smart Search queries can be used for many things, and targeting is one of them (in the context of tasks and call assignments). As long as we're only dealing with tasks, target is synonymous with query. In another context, e.g. regular standalone Smart Search queries, referring to them using the "target" terminology can at worst be confusing.

In the context of call assignments however, there are two Smart Search queries, one of which is the target, and the other is called the "goal". In that context, referring to the goal query as the "target" is downright misleading. So we should refrain from using the "target" terminology in any place where it isn't specifically and exclusively the type of Smart Search that we are working with.

A short fruit analogy: Apples and bananas are both fruits, but fruits aren't necessarily bananas or apples. 😊 

Conclusion: I think `targetId` should be renamed to `queryId`.